xml/ay_firewall.xml

```diff
@@ -340,6 +340,21 @@
     </informaltable>
    </sect2>
 
+   <sect2 xml:id="CreateProfile-firewall-stage">
+      <title>When the Firewall Profile is Applied</title>
+
+      <para>
+       The profile is completely applied at the end of the first stage of installation, except when 
+       the installation runs over the network, for example over SSH or VNC. In this case we have 
+       to guarantee that the installed system stays accessible for the whole installation process. For 
+       that reason the firewall configuration is postponed to the second stage. If you need the network 
+       for some reason to fulfill installation successfully, you have to take care of proper firewall 
+       configuration. Last note is for cases when firewall configuration is not present in 
+       the profile at all. Firewall configuration is done according to the product settings as in common
+       installation in such case.
```

Comment:
"Firewall configuration is done according to the product settings as in common       installation in such case." Does this mean that in the absence of a profile, there are default firewall settings that are applied?

thanks!

Reply:
Not only in case of absence of a profile, but always when firewall section is missing.